src/Utils/dbConnection.ts

```diff
@@ -0,0 +1,16 @@
+import { getConnectionOptions, createConnection } from "typeorm";
```

Comment:
In my latest PR, to solve test issues. connection creation is in the `server.ts` file